### PR TITLE
Importing backups

### PR DESCRIPTION
**Pull request by dominikwilkowski**
WIP

### CODE REVIEW

File: vault_db.toml
Comment (thewizzy) [inline vault_db.toml:1]: Are these changes necessary?

**Review by thewizzy — DISMISSED** [review]

File: vault_db.toml
Comment (dominikwilkowski) [reply vault_db.toml:1]: Yeah I added them because it means we can test more. Before I had only one entry that had multiple history entries which was lacking and really obvious in the import flow

**Review by thewizzy — APPROVED** [review]
